TLSharp.Core/TelegramClient.cs

```diff
@@ -28,8 +28,10 @@ public class TelegramClient : IDisposable
         private int _apiId = 0;
         private Session _session;
         private List<TLDcOption> dcOptions;
+        private TcpClientConnectionHandler _handler;
```

Comment:
Only one last thing: is this field really needed? it seems you only use it to pass it to the TcpTransport constructor.